src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx

```diff
@@ -77,137 +63,205 @@ helm upgrade --install newrelic-bundle newrelic/nri-bundle \
 To install the Kubernetes agent operator with the default configuration, run these commands:
 
 ```shell
-helm repo add newrelic-k8s https://newrelic.github.io/k8s-agents-operator
-helm upgrade --install k8s-agents-operator newrelic-k8s/k8s-agents-operator \
+helm repo add k8s-agents-operator https://newrelic.github.io/k8s-agents-operator
+helm upgrade --install k8s-agents-operator k8s-agents-operator/k8s-agents-operator \
     --namespace newrelic \
     --create-namespace \
     --set licenseKey=YOUR_NEW_RELIC_INGEST_LICENSE_KEY
 ```
 
 For a complete list of configuration options, see the [README](https://github.com/newrelic/k8s-agents-operator/tree/main/charts/k8s-agents-operator#values) chart.
 
-## Configuring a namespace to enable the operator [#configuring-namespace]
+## Monitored namespaces [#monitored-namespaces]
 
-Each namespace requires a two-step process to enable automatic instrumentation of applications deployed within it.
+A Kubernetes secret will automatically be replicated from the operator's namespace into all namespaces that monitor a pod. Inside the CRD YAML file, specify which APM agents you want to instrument. All available APM agent docker images and corresponding tags are listed on DockerHub:
 
-1. Create a secret containing a valid [New Relic ingest license key](/docs/apis/intro-apis/new-relic-api-keys/#license-key) for each namespace you want the operator to instrument.
+    * [Java](https://hub.docker.com/repository/docker/newrelic/newrelic-java-init/general)
+    * [Node](https://hub.docker.com/repository/docker/newrelic/newrelic-node-init/general)
+    * [Python](https://hub.docker.com/repository/docker/newrelic/newrelic-python-init/general)
+    * [.NET](https://hub.docker.com/repository/docker/newrelic/newrelic-dotnet-init/general)
+    * [Ruby](https://hub.docker.com/repository/docker/newrelic/newrelic-ruby-init/general)
 
-   ```shell
-   kubectl create secret generic newrelic-key-secret \
-       --namespace NAMESPACE_TO_MONITOR \
-       --from-literal=new_relic_license_key=YOUR_NEW_RELIC_INGEST_LICENSE_KEY
-   ```
+For .NET   
+```yaml
+apiVersion: newrelic.com/v1alpha2
+kind: Instrumentation
+metadata:
+  name: newrelic-instrumentation-dotnet
+spec:
+  agent:
+    language: dotnet
+    image: newrelic/newrelic-dotnet-init:latest
+    # env: ...
+```
+For Java
 
-   Make sure to replace `YOUR_NEW_RELIC_INGEST_LICENSE_KEY` with your valid New Relic license key.
+```yaml
+apiVersion: newrelic.com/v1alpha2
+kind: Instrumentation
+metadata:
+  name: newrelic-instrumentation-java
+  namespace: newrelic
+spec:
+  agent:
+    language: java
+    image: newrelic/newrelic-java-init:latest
+    # env: ...
+```
 
-2. Apply the following Custom Resource Definition (CRD) to each namespace you want to auto instrument. Run this command:
+For NodeJS
```

Comment:
@jeff-colucci Seems odd not to have colons after lines like this.

Also, it should be "Node.js"